homeworks/olexandr.reshetnyk_reshetnyk/homework_6/task_1.js

```diff
@@ -0,0 +1,29 @@
+function addUser() {
+    let fullName;
+    let phone;
+    const obj = {};
+    Object.defineProperty(obj, 'fullName', {
+        get() {
+            return fullName;
+        },
+        set(value) {
+            fullName = value.toLowerCase().replace(/\b\w/g, firstLetter => firstLetter.toUpperCase());
+        },
+    });
+    Object.defineProperty(obj, 'phone', {
+        get() {
+            return phone;
+        },
+        set(value) {
+            phone = value.toLowerCase().replace(/(^\+)|\D*(\d+)\D*/g, (str, plus, num) => (plus) || num);
```

Comment:
hmmm, looks like easier here is to look for non-digits first and then add plus sign optionally